### DISCUSSION TITLE
Questions about listen_bucket_notification in Python API

### DISCUSSION THREAD

**Discussion by Snolte1001** [minio/minio#13099] [Q&A] [answered]
Hello

is it possible to get notified with the Python API when a new bucket is created ? 
With "listen_bucket_notification" I only succeeded in getting a notification when something was changed in an already existing bucket.

second question:
 Is it possible to queue missed events so that if my python script is not running, they are not lost ? 

third question:
 Is it possible to use a regular expression for the prefix of the bucket notification ?

**Comment by balamurugana** [comment, marked as answer]
1. No. There is no such support.
2. You would need to have your own service to store events for later usage.
3. No.  There is no such support in S3 specification.
